api/mapping/services_rules.py

```diff
@@ -1,9 +1,288 @@
-from data.models import Concept
+from data.models import Concept, ConceptRelationship
 
-from mapping.models import ScanReportConcept
+from mapping.models import ScanReportConcept, OmopField, Concept, StructuralMappingRule
 from mapping.serializers import ConceptSerializer
 
 
+class NonStandardConceptMapsToSelf(Exception):
+    pass
+
+           
+#allowed tables
+m_allowed_tables = ['person','measurement','condition_occurrence','observation']
+
+#names of date objects in ScanReportTable
```

Comment:
We are rollbacking this approach. According to the new standard's doc, each table can have only one date field. So, if a table contains conditions and observations, both must use the same date column.

Reply:
ok, but since this has not yet been rolled back, is there much I can do here?

or should it become something like this:
```
    'condition_occurrence':'observation_date',
    'observation':'observation_date'
```

Reply:
https://dev.azure.com/UniversityOfNottingham/DRS/_backlogs/backlog/CO-CONNECT/Epics/?workitem=35352